doc/PROFICIENCY.md

```diff
@@ -0,0 +1,19 @@
+# proficiency
+
+## definition
+
+```JSON
+{
+  "id": "prof_knapping",
+  "type": "proficiency",
+  "name": { "str": "Knapping" }
+},
+```
+### `id`
+Mandatory. The id of the proficiency.
+
+### `type`
+Mandatory. Must be `proficiency`.
+
+### `name`
+Mandatory. The name of the proficiency.
```

Comment:
This might fit better in `JSON_INFO.md` rather than its own new file?  I'm not sure if we really have any agreed principles for organisation of documentation, so I guess this is personal preference.

Reply:
I plan on adding more to this, and personally am not a huge fan of the single massive documentation file covering everything.